### PR TITLE
[REG-1602] Docs on the GPT Assistant Bot

### PR DESCRIPTION
**Pull request by analogrelay**
This PR adds a guide on how to configure the GPT Assistant Bot in your game

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

**Review by abeizer — APPROVED** [review]
Love the use of gifs for these docs. My bot struggled to host a game and select a character, but the fact GPT can click through menus and that we have a decent starting point for expanding on the idea is very neat.

File: docs/automated-testing/gpt-assistant-bot.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: GPT Assistant Bot
+---
+
+# Using the GPT Assistant Bot
+
+In addition to the completely random "Monkey" Bot, the RegressionGames Automated UI Testing package includes a "GPT Assistant" Bot that uses [OpenAI's GPT-4 Large Language Model](https://openai.com/gpt-4) to drive a bot in your game.
+This guide will show you how to set up the GPT Assistant Bot, how to connect it to OpenAI, and how to give the bot a goal to accomplish.
+
+### Prerequisites
```
[the remark this answers is not in the document]
Comment (analogrelay) [reply docs/automated-testing/gpt-assistant-bot.mdx:10]: Oop, good catch. I did some reheadering and probably missed some. I'll rescan and fix them up.

[the remark this answers is not in the document]
Comment (analogrelay) [reply docs/automated-testing/gpt-assistant-bot.mdx]: Fixed!

**Review by RG-nAmKcAz — APPROVED** [review]

**Review by addisonbgross — APPROVED** [review]
